docs/howtos/raw-policies.md

```diff
@@ -0,0 +1,274 @@
+---
+sidebar_label: "Raw policies"
+title: "Raw policies"
+---
+
+# Raw policies
+
+From `v1.9.0` onwards, Kubewarden supports the ability to write and execute policies
+outside a Kubernetes cluster, as a generic policy evaluation engine.
+The Policy Server exposes the `/validate_raw` endpoint that can be used to validate
+arbitrary JSON documents against a Kubewarden policy.
+
+For this guide, we will use the following raw policies:
+
+- [raw-validation-policy](https://github.com/kubewarden/raw-validation-policy)
+- [raw-mutation-policy](https://github.com/kubewarden/raw-mutation-policy)
+
+:::note
+Please ensure that the policy author has marked the policy as `policyType: raw` in the metadata.
+You can inspect the metadata by using `kwctl`
+
+```bash
+kwctl inpect ghcr.io/kubewarden/tests/raw-mutation-policy:v0.1.0
+```
+
+:::
+
+## Running the policy server outside Kubernetes
+
+The Policy Server can be run outside Kubernetes as a standalone container.
+
+First, create a `policies.yml` file with the following content:
+
+```yaml
+raw-validation:
+  url: ghcr.io/kubewarden/tests/raw-validation-policy:v0.1.0
+  settings:
+    validUsers:
+      - alice
+      - bob
+    validActions:
+      - read
+      - write
+    validResources:
+      - orders
+      - products
+
+raw-mutation:
+  url: ghcr.io/kubewarden/tests/raw-mutation-policy:v0.1.0
+  allowedToMutate: true
+  settings:
+    forbiddenResources:
+      - privateResource
+      - secretResource
+    defaultResource: publicResource
+```
+
+To start the policy server:
+
+```bash
+# Create a docker volume to store the policies
+docker volume create --driver local \
+                --opt type=tmpfs \
+                --opt device=tmpfs \
+                --opt o=ui=65533 \
+                policy-store
+
+# Start the policy server
+docker run --rm -it \
+    -p 3000:3000 \
+    -v $(pwd)/policies.yml:/policies.yml \
+    -v policy-store:/registry \
+    ghcr.io/kubewarden/policy-server:1.9.0 \
+    --ignore-kubernetes-connection-failure=true
+```
+
+NOTE: the flag `--ignore-kubernetes-connection-failure=true` is required to start the policy server without Kubernetes.
+However, it is possible to start the Policy Server with/inside Kubernetes and use the raw validation endpoint.
+Raw policies can access context-aware[ capabilities](../explanations/context-aware-policies.md) like standard policies.
+
+## Running a Policy Server inside Kubernetes without the Kubewarden controller
+
+Alternatively, the Policy Server can be deployed in Kubernetes without the Kubewarden controller.
```

Comment:
Nothing prevents the user from using the `validate_raw` endpoint of a controller-managed Policy server. The goal of this part of the how-to is to document alternative installation methods since the user might or might not want to install the full Kubewarden stack via the helm charts.

Reply:
No this is not possible, the problem is subtle. To host a raw policy the user must change the `policies.yml` file loaded by the Policy Server instance. However, this "file" is managed by our controller. Hence any change done to this file will then be reverted by our controller.

I wonder if we should ad that to the suggested text :thinking:

Reply:
Ah yes i totally missed that! I will apply the suggestion thanks!